### ISSUE TITLE
Warning when config provided

### ISSUE THREAD

**Issue by plicease** [plicease/Test-Alien-CPP#3]
This is a warning on my os x with clang++, but it might be an error on other platforms.

```
# $VAR1 = {
#           'extra_linker_flags' => ' -lstdc++',
#           'config' => {
#                         'cc' => 'clang++'
#                       },
#           'extra_compiler_flags' => ' -fno-common -DPERL_DARWIN -no-cpp-precomp -mmacosx-version-min=10.13 -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -DPERL_USE_SAFE_PUTENV -xc++ -Wno-reserved-user-defined-literal'
#         };
# 
# 
# 
t/00_diag.t ................... ok   
t/01_use.t .................... ok   
t/test_alien_cancompilecpp.t .. ok   
t/test_alien_cpp.t ............ extra Module::Build option: config at /Users/ollisg/dev/Test-Alien-CPP/lib/Test/Alien/CPP.pm line 105.
extra Module::Build option: config at /Users/ollisg/dev/Test-Alien-CPP/lib/Test/Alien/CPP.pm line 105.
extra Module::Build option: config at /Users/ollisg/dev/Test-Alien-CPP/lib/Test/Alien/CPP.pm line 105.
t/test_alien_cpp.t ............ ok   
t/test_alien_cpp__gh1.t ....... 1/? extra Module::Build option: config at /Users/ollisg/dev/Test-Alien-CPP/lib/Test/Alien/CPP.pm line 105.
t/test_alien_cpp__gh1.t ....... ok   
All tests successful.
Files=5, Tests=7,  3 wallclock secs ( 0.03 usr  0.02 sys +  1.94 cusr  0.60 csys =  2.59 CPU)
Result: PASS
```

**Comment by plicease** [comment]
I think this is due to a recent change in ExtUtils::CppGuess.  Probably an easy fix, but I don't have time to work on it right this minute, and may need updates to Test::Alien.

**Comment by plicease** [comment]
Fixed with #5
